src/templates/SearchLibrary.jsx

```diff
@@ -165,7 +158,7 @@ const SearchLibrary = ({
     }
     if (currentCatalog) {
       currentCatalog.forEach((element) => {
-        url += "catalog=" + element + "&";
+        url += "catalog_version=" + element + "&";
```

Comment:
Question: Not to be handled in this PR but how sure are we about the validity of `url` here?  It seems like an easy situation for creating a bug when the `url` `:string` is missing the trailing `/`.

Reply:
Oh gosh- I think this whole section could use a refactor. It wasn't easy to understand and there aren't tests for it. The change here was as minimal as possible, but I hope this code can be revisited in a bigger way in the future.